boxoffice/templates/boxoffice.html

```diff
@@ -0,0 +1,619 @@
+            <div class="col-md-6 col-md-offset-3 col-sm-8 col-sm-offset-2">
+              <input type="text" class="form-control buyer-fields secondary" name="phone" value = {{buyer.phone }} placeholder="Phone"/>
+            </div>
+          </div>
+          <div class="checkout-confirmation col-md-6 col-md-offset-3 col-sm-8 col-sm-offset-2">
+            <p class="summary">Booking Summary</p>
+            {{#order.line_items}} {{# { line_item: . } }}
+              {{#if line_item.quantity > 0 }}
+                <div class="item-summary">
+                  <p class="booked-ticket-title"><i class="fa fa-ticket"></i>{{ line_item.item_title }}</p>
+                  <p class="booked-ticket-qty">Qty: {{ line_item.quantity }} &times; <i class="fa fa-inr"></i> {{line_item.base_price}}</p>
+                  {{#line_item.discounted_amount}}
+                    <p class="booked-ticket-qty">Discount: <i class="fa fa-inr"></i> {{line_item.discounted_amount}}</p>
+                  {{/line_item.discounted_amount}}
+                  {{#line_item.discount_policies}}
+                    {{#if activated}}
+                    <p class="discount-applied">Applied: {{ title }}</p>
+                    {{/if}}
+                  {{/line_item.discount_policies}}
+                  <p class="booked-ticket-qty">Subtotal: <i class="fa fa-inr"></i> {{line_item.final_amount}}</p>
+                </div>
+              {{/if}}
+            {{/}}{{/order.line_items}}
+            <p class="price">Amount Payable: <span class="final-amount"><b><i class="fa fa-inr"></i> {{ order.final_amount }}</b></span></p>
+          </div>
+          <div class="proceed-button-wrapper">
+            <button class="boxoffice-button boxoffice-button-info edit-order" on-click="selectItems(event)">Edit order</button>
+            <button type="submit" class="boxoffice-button boxoffice-button-action">
+              Checkout {{#loadingOrder}}<i class="fa fa-spinner fa-spin" intro='fly:{"x":0,"y":"0"}'>{{/}}
+            </button>
+          </div>
+          {{#errorMsg}}
+            <p class="stage-error-msg text-danger">{{ errorMsg }}</p>
+          {{/}}
+        </form> 
+      </div>
+    {{elseif id === 'boxoffice-confirm' && id === activeTab}}
+      <div id='{{id}}' class="boxoffice-section clearfix" intro='fly:{"x":600,"y":"0"}'>
+        {{#section}}
+          <p class="confirmation-icon"><i class="fa fa-check-circle"></i></p>
+          <p class="confirmation-header">Thank you for your order!</p>
+          <div class="confirmation-msg">
+            <p class="name">Hello {{buyer.name}},</p>
+            <p>We have sent you a confirmation email with the <a href="{{ cashReceiptURL }}" target="_blank" class="boxoffice-button boxoffice-button-info receipt-button" on-click="fire('eventAnalytics','view cash receipt', 'cash receipt')">cash receipt</a> <br>We will get in touch with you shortly for the attendee details.</p>
```

Comment:
Could directly link to the "cancellation and refund policy" subsection of the site policies page.